opm/simulators/flow/FlowMainEbos.hpp

```diff
@@ -480,10 +484,10 @@ namespace Opm
             const auto& timeMap = schedule.getTimeMap();
             auto& ioConfig = eclState().getIOConfig();
             SimulatorTimer simtimer;
-
+            size_t end_step = EWOMS_GET_PARAM(TypeTag, int, EndStep);
```

Comment:
The declared parameter is an `int`, but you're assigning it to a `size_t`.  What is the right parameter type here?